.gitignore

```diff
@@ -1,3 +1,38 @@
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+crash.*.log
+
+# Exclude all .tfvars files, which are likely to contain sensitive data, such as
+# password, private keys, and other secrets. These should not be part of version
+# control as they are data points which are potentially sensitive and subject
+# to change depending on the environment.
+*.tfvars
+*.tfvars.json
+
+# Ignore override files as they are usually used to override resources locally and so
+# are not checked in
+override.tf
+override.tf.json
+*_override.tf
+*_override.tf.json
```

Comment:
What is the source of this? If you don't use it, remove because they are likely to introduce issues with ad-hoc fixes that are not checked in.

Reply:
THe source of the .gitignore content is the official Terraform one : https://github.com/github/gitignore/blob/main/Terraform.gitignore

Reply:
We don't need it. You can remove.

Reply:
Done